TAO/orbsvcs/tests/Security/ssliop_corbaloc/client.cpp

```diff
@@ -1071,5 +1078,6 @@ ACE_TMAIN(int argc, ACE_TCHAR *argv[])
   if (cosnaming_client.init (argc, argv) == -1)
     return 1;
 
+  cosnaming_client.activate();
```

Comment:
When you add threading the main should use the ACE thread manager to wait on the worker threads to be finished to not get strange crashes at shutdown